oms-all-deploy/nestedtemplates/omsWorkspace.json

```diff
@@ -179,7 +179,7 @@
           },
           "properties": {
             "ETag": "*",
-            "query": "Type=Event EventID=40 OR EventID=36� Source=\"DISK\" EventLog=\"System\" TimeGenerated>NOW-24HOURS | Measure Count() By Compute",
+            "query": "Type=Event EventID=40 OR EventID=36� Source=\"DISK\" EventLog=\"System\" TimeGenerated>NOW-24HOURS | Measure Count() By Compute",
```

Comment:
This renders as a non-displayable character in the PR UI; what character is this?